ByteDance/CHANGELOG.md

```diff
@@ -1,7 +1,8 @@
 # Changelog
 
-## 4.8.1.0.3
-* Fix native ad view ad not triggering show and click event callbacks.
```

Comment:
Looks like you accidentally deleted an entry from the Changelog

Reply:
Yeah I noticed that, but it was also deleted on the iOS-SDK-Mediation-Adapters repo, https://github.com/AppLovin/iOS-SDK-Mediation-Adapters/pull/722/files#diff-0f02b5b4454786ab661a67502e07287b3b340de556d2527a6311f9a3dcc9c965